README.md

```diff
@@ -25,6 +25,30 @@ TiKV uses Raft to support Geo-Replication. We have ported etcd's raft implementa
 
 This project requires rust nightly, otherwise it can't build.
 
+```
+# 1. Install Rust Nightly
+curl -s https://static.rust-lang.org/rustup.sh | sh -s -- --channel=nightly
+
+# 2. Install RocksDB Library
+
+wget https://github.com/facebook/rocksdb/archive/rocksdb-4.3.1.tar.gz
+tar -xzvf rocksdb-4.3.1.tar.gz
+
+# OR
+git clone https://github.com/facebook/rocksdb.git
+
+cd rocksdb
+make shared_lib
+cp cp librocksdb.so* /usr/lib64/
```

Comment:
@overvenus 

I haven't 32bit Linux OS for test. Maybe copy to  `lib` folder:

`cp librocksdb.so* /usr/lib/`